package-system/assimp/Findassimplib.cmake

```diff
@@ -30,20 +30,20 @@ set(${LIB_NAME}_BIN_DIR ${CMAKE_CURRENT_LIST_DIR}/assimp/bin)
 set(${LIB_NAME}_LIBS_DIR ${CMAKE_CURRENT_LIST_DIR}/assimp/lib)
 
 if (${PAL_PLATFORM_NAME} STREQUAL "Linux")
-    set(${LIB_NAME}_LIBRARY_DEBUG   ${${LIB_NAME}_BIN_DIR}/libassimp.so.5.1.6)
-    set(${LIB_NAME}_LIBRARY_RELEASE ${${LIB_NAME}_BIN_DIR}/libassimp.so.5.1.6)
+    set(${LIB_NAME}_LIBRARY_DEBUG   ${${LIB_NAME}_BIN_DIR}/libassimp.so.5.2.4)
```

Comment:
[](https://github.com/assimp/assimp/blob/v5.2.5/CMakeLists.txt )

the 5.2.5 tag outputs shared libraries with 5.2.4 which is sort of a AssImp bug. The 3p package (and Tag) are both still 5.2.5

Reply:
Couldn't you add a fix to the .patch file to output the correct 5.2.5 for every library?

Reply:
Well, I was using the patch file for fixing compile issues. This is more of a minor issue that does not really affect the release of the package. If you feel it is important enough to stop releasing this version then it can be done.

Reply:
My concern is it will be confusing to see two versions over in the O3DE engine code that links these libraries. Someone not familiar with this gotcha may wonder why we're using an older version of the library on some platforms. It's not a self documenting issue, and I could see us forgetting about this in six months.

I'll give a ship it and you can decide if you want to make the change or not.

Reply:
I pointed this out to our 3p package library experts and they did not seem to be concerned about it. But really I can go either way on this. The only thing this change risks is a slight delay of delivering the packages (like January 2023).